Packs/TaniumThreatResponse/Integrations/TaniumThreatResponseV2/TaniumThreatResponseV2.py

```diff
@@ -254,6 +258,13 @@ def get_event_header(event_type):
     return headers
 
 
+''' GENERAL HELPER FUNCTIONS'''
+
+
+def normalize_api_response(raw_response):
```

Comment:
Please add a docstring explaining this was added to support the difference between v4 and v3.